.ruby-version

```diff
@@ -1 +1 @@
-3.0.0
+3.1.3
```

Comment:
Would bump to 3.1.4
```suggestion
3.1.4
```

Also not sure if would be benefit to going all the way to Ruby 3.2 at this point

Reply:
Ruby 3.2 has some nice additions.  Is there a reason not to upgrade to the current version?